src/d3a/models/config.py

```diff
@@ -110,3 +108,6 @@ def read_market_maker_rate(self, market_maker_rate):
         self.market_maker_rate = read_arbitrary_profile(InputProfileTypes.IDENTITY,
                                                         market_maker_rate_parsed)
         self.market_maker_rate = {k: float(v) for k, v in self.market_maker_rate.items()}
+
+    def read_iaa_fee(self, iaa_fee):
```

Comment:
I thought we agreed on a more general function like `read_config_params` or so that incorporates all parameters of the SimulationConfig class. This way the user does not have to call multiple functions to set the parameters but only one. Or did i get i wrong?

Reply:
How about updating it via already existed method(https://github.com/gridsingularity/d3a/pull/415/files#diff-ab2b09a6e41de97909debee94cf77eb1R83)?

Reply:
This could work (although this gets called every tick and might be a bit of an overkill)

Reply:
Okay, since the rest of parameters are explicitly updated via their own method. I suggest its better to have a separate method for iaa_fee too. But I have no strong opinion. Let me know, what looks good to you guys.

Reply:
I agree with @faizan2590 , the event_update method is already used for area events (hence the 'event' part of the method), therefore it looks good to my eyes to not have a separate function to update these parameters via the setup file, but to reuse the same function. It is easy to adapt it and add the iaa_fee and market maker rate to it, even if the area events do not use these yet. 
I would vote against having a separate method for setting the IAA fee, because I find it more convenient to have one method that can set these parameters at once instead of having to call 2 methods.

Reply:
Ok, makes sense, voting also for adding it to `event_update` (and also rename this function)

Reply:
Sorry, I didn't get the reason to rename the method name

Reply:
Just for the user. I guess it is confusing to type `conf.event_update()`  but really want to set some configuration parameter. Does it make sense?

Reply:
So, any update here? I see that you did not rename it.

Reply:
Sorry, missed it!